staging/src/k8s.io/apiextensions-apiserver/test/integration/objectmeta_test.go

```diff
@@ -308,20 +308,6 @@ properties:
       kind: Pod
       labels:
         foo: bar
-  invalidDefaults:
```

Comment:
why was this case dropped? I thought this would continue to be invalid

Reply:
because we cannot mix positive and negative tests now that we validate everything under metadata.